src/main.rs

```diff
@@ -1,112 +1,177 @@
-use clap::Parser;
-use std::fs::{self, File};
+use clap::{Parser, Subcommand};
+use std::{
+    fs::{self, File},
+    process::Command,
+};
 
 /// Command line tool for C projects
 #[derive(Parser, Debug, Clone)]
 #[command(version, about, long_about = None)]
 struct Args {
     /// Name of the project
-    project_name: String,
+    // project_name: Option<String>,
+    #[command(subcommand)]
+    command: Option<Commands>,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+enum Commands {
+    /// Create new C Proejct with name
+    CNew {
```

Comment:
Agree with clippy here, I think prefixes in Enum variants are redundant.

Please remove the `C` prefix for each variant.

The project itself is named `cinit` and in the description/Readme describes that it is creating a C language tool, so it is not needed to prefix with `c` in functions variants and whatever.